src/utils/update/trust.model.fetch.ts

```diff
@@ -319,7 +319,9 @@ export const mapTrusteeType = (trusteeTypeId: string): RoleWithinTrustType => {
       case "5002":
         return RoleWithinTrustType.BENEFICIARY;
       case "5001":
-        return RoleWithinTrustType.BENEFICIAL_OWNER;
+        // Type 5001 is not a valid type for an individuual or corporate trustee
+        logger.info(`Warning - invalid data. Trustee type ${trusteeTypeId} found when mapping trustee data`);
```

Comment:
Guessing 'info' is the right log level. We don't seem to have a 'warning' level and 'error' might result in lots of log entries due to dodgy spreadsheet data.

Perhaps worth updating the comment to say why this might occur? Dodgy spreadsheet data...

Reply:
i'd say info is ok, as it's an expected scenario?

Reply:
👍 still think a comment explaining why we're having to handle the invalid type would be useful.

Reply:
Comment updated